.buildkite/pipeline.yml

```diff
@@ -697,3 +697,57 @@ steps:
     trigger: "beats-agent-serverless-tests"
     build:
       branch: "main"
+
+  - label: "Trigger DRA (SNAPSHOT)"
+    if: build.pull_request.id != null
+    plugins:
+      - monorepo-diff#v1.0.1:
+          diff: "git diff --name-only origin/${GITHUB_PR_TARGET_BRANCH}...HEAD"
+          interpolation: false
+          watch:
+            - path:
+                - .go-version
+                - .buildkite/packaging.pipeline.yml
+                - .buildkite/pipeline.yml
+                - .buildkite/scripts/dra.sh
+                - .buildkite/scripts/packaging/
+                - dev-tools/packaging/
+              config:
+                trigger: "beats-packaging-pipeline"
+                build:
+                  commit: "${BUILDKITE_COMMIT}"
+                  branch: "${BUILDKITE_BRANCH}"
+                  env:
+                    - BUILDKITE_PULL_REQUEST=${BUILDKITE_PULL_REQUEST}
+                    - BUILDKITE_PULL_REQUEST_BASE_BRANCH=${BUILDKITE_PULL_REQUEST_BASE_BRANCH}
+                    - GITHUB_PR_LABELS=${GITHUB_PR_LABELS}
+                    - DRA_BRANCH=${BUILDKITE_PULL_REQUEST_BASE_BRANCH}
+                    - RUN_SNAPSHOT=true
+                    - ELASTIC_SLACK_NOTIFICATIONS_ENABLED=false
+                    - DRY_RUN=true
+
+  - label: "Trigger DRA 7.17 Elastic Agent (SNAPSHOT)"
```

Comment:
I wonder if this section should be added only in the backport PR to `7.17` rather than on `main` and active-9/active-8 as keeping it here in so many branches will be no-op code.

WDYT?

Reply:
also same comment as earlier might be beneficial to say `Trigger DRA (SNAPSHOT) in dry-run mode" or something similar